crt_portal/api/views.py

```diff
@@ -102,3 +103,15 @@ class ReportCountView(APIView):
     def get(self, request, format=None):
         reports_accessed_payload = reports_accessed_filter(request.GET)
         return Response(reports_accessed_payload)
+
+
+class FormLettersIndex(APIView):
+    """
+    A view that displays information about the number of form letters sent.
+    """
+
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        contacts_payload = contacts_filter(request.GET)
```

Comment:
todo: add error handling for when a user uses an invalid date format.  Right now, an invalid date format causes all results to be returned.